flit/init.py

```diff
@@ -224,7 +224,6 @@ def initialise(self):
                 f.write(tomli_w.dumps({'readme': readme}))
             if license != 'skip':
                 f.write(tomli_w.dumps({'license': license_names_to_spdx[license]}))
-                f.write(f"license-files = {json.dumps(['LICENSE'])}\n")
```

Comment:
Adding a dedicated `license-files = ['LICENSE']` line to `pyproject.toml` does have the benefit that it's directly visible to the user. No need to lookup what the exact key is.

I'd suggest to keep it as is.

Reply:
Fair enough, my main complaint was that this didn't allow for an extension, so perhaps?:

Reply:
> Fair enough, my main complaint was that this didn't allow for an extension

Does it need to though? I believe the value here is designed to match the initial config. If the license file name is changed, flit would also emit an error as the pattern didn't match any file, which would lead the user to update it.